src/lib/constants.ts

```diff
@@ -26,9 +26,11 @@ export const servicesObject: Record<string, string> = {
 	knightcrawler: 'Knightcrawler',
 	orionoid: 'Orionoid',
 	jackett: 'Jackett',
+	realdebrid: 'Real Debrid',
+	alldebrid: 'All Debrid',
 	torbox: 'Torbox',
 	mediafusion: 'Media Fusion',
-	Prowlarr: 'Prowlarr',
+	prowlarr: 'Prowlarr',
```

Comment:
This causes Prowlarr to display as undefined on the summary page. It should be capitalized but I agree that we should have an uniform naming schema. Maybe change the `/stats` endpoint on the backend PR to fix this typo?

![image](https://github.com/user-attachments/assets/6bdaed04-a024-4ffd-b871-f4bbbb8672c4)

Reply:
Had to add a small commit to fix this :)